images/oai/Dockerfile

```diff
@@ -19,48 +19,36 @@ RUN apt-get update && \
        git \
        xxd \
        #unzip is needed for protobuf
-       unzip
+       unzip \
+       # python3-pip for conf template generation
+       python3-pip && \
+    pip3 install --ignore-installed pyyaml
 
 # In some network environments, GIT proxy is required
 RUN /bin/bash -c "if [[ -v NEEDED_GIT_PROXY ]]; then git config --global http.proxy $NEEDED_GIT_PROXY; fi"
 
-RUN apt-get update && apt-get install -y git
 #create the WORKDIR
 WORKDIR /oai-ran
 RUN git clone https://gitlab.eurecom.fr/oai/openairinterface5g.git . 
 RUN git checkout $VERSION
 
-RUN sed -i 's/libuhd4.2.0/libuhd4.3.0/g' cmake_targets/tools/build_helper
-
 #run build_oai -I to get the builder image
 RUN /bin/sh oaienv && \ 
     cd cmake_targets && \
     mkdir -p log && \
-    ./build_oai -I -w USRP
+    ./build_oai -I -w USRP --install-optional-packages
 
 #run build_oai to build the target image
 RUN /bin/sh oaienv && \ 
     cd cmake_targets && \
     mkdir -p log && \
-    ./build_oai --eNB -w USRP --verbose-ci
-
-
+    ./build_oai --eNB --gNB --RU --UE --nrUE --ninja --build-lib "telnetsrv enbscope uescope nrscope nrqtscope" -w USRP --verbose-ci --noavx512 -c
```

Comment:
We should also be able to create the avx512 variant with this Dockerfile. Set `ARG build_flags="--noavx512 -w USRP` as a build-arg to be able to build the avx512 variants with:
`docker build --build-arg build_flags="-w USRP" -t ... `